include/swift/AST/DiagnosticsSema.def

```diff
@@ -1121,7 +1121,10 @@ ERROR(missing_dynamic_callable_kwargs_method,none,
 ERROR(invalid_dynamic_member_lookup_type,none,
       "@dynamicMemberLookup attribute requires %0 to have a "
       "'subscript(dynamicMember:)' method that accepts either "
-      "'ExpressibleByStringLiteral' or a keypath", (Type))
+      "'ExpressibleByStringLiteral' or a key path", (Type))
+NOTE(invalid_dynamic_member_subscript, none,
+     "add an explicit argument label to this subscript to satisfy "
+     "the @dynamicMemberLookup requirement", ())
```

Comment:
Not a blocker: how about enclosing `'@dynamicMemberLookup'` within single quotes in diagnostic messages?

It would be nice to standardize this. Currently, it's done for in some diagnostics (`'@objc'`, `'@usableFromInline'`, `'@frozen'`, etc) but not others (`@dynamicMemberLookup`, `@dynamicCallable`, etc).

Reply:
Good idea! I’ll open a new PR to do this to cover other attributes as well, not just `@dynamicMemberLookup`.